Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/MicrosoftGraphMail.py

```diff
@@ -56,6 +56,14 @@
     'sent items': 'sentitems',
 }
 
+GRAPH_BASE_ENDPOINTS = {
```

Comment:
why dont we do it in the api module? isnt that something all graph integration will need?

Reply:
It won't be the same for all graph integrations because not all graph integrations have a field of `host_url` in the instance configuration. Still, I moved it to the apiModule as it is common for at least several integrations.